src/main/kotlin/Decorator/littlekai/base/Noodles.kt

```diff
@@ -1,5 +1,6 @@
 package Decorator.littlekai.base
 
-interface  Noodles {
-  fun calculateCost() : Double
+interface Noodles {
+  fun calculateCost(): Double
+  fun calculateTotalCost(): Double
```

Comment:
I think with the fact of implementing the decorator function in each concrete case, it's already adding behaviour. What do you think about it?